From 855ecb8b9bf5440842d2ee8bcd8e8fda80ba23ea Mon Sep 17 00:00:00 2001
From: "sungmok.shin" <sungmok.shin@lge.com>
Date: Thu, 20 Sep 2018 13:52:52 +0900
Subject: [PATCH] Build: Do not purge build-scripts after build

Forced re-compile of iotivity-node after successful build introduces
compile error lack of build-scripts/postinstall.js. Currently,
postinstall.js erases build-scripts folder after successful build.
This patch let postinstall.js not erase the build-scripts folder to
prevent force re-compile of iotivity-node in webOS OSE.

Upstream-Status: Pending
---
 build-scripts/postinstall.js | 1 -
 1 file changed, 1 deletion(-)

diff --git a/build-scripts/postinstall.js b/build-scripts/postinstall.js
index 4519bea..2b78d64 100644
--- a/build-scripts/postinstall.js
+++ b/build-scripts/postinstall.js
@@ -61,7 +61,6 @@ shelljs.rm( "-rf",
 	path.join( repoPaths.root, ".gitattributes" ),
 	path.join( repoPaths.root, "node_modules", "nan" ),
 	path.join( repoPaths.root, "node_modules", ".bin" ),
-	path.join( repoPaths.root, "build-scripts" ),
 	path.join( repoPaths.installLibraries, "*.a" ),
 	path.join( repoPaths.installLibraries, "*.lib" ),
 	path.join( repoPaths.installLibraries, "*.dll" ),
-- 
1.9.1

